general/administration/hardware-acceleration.md

```diff
@@ -151,10 +151,11 @@ Install linux-headers and run the following command.
 apt-get install linux-headers-$(uname -r | sed 's/[^-]*-[^-]*-//')
 ```
 
-Alternatively, run this command if you are on stretch for compatibility.
+Alternatively, run thoses commands if you are on stretch for compatibility.
 
 ```sh
-apt-get install -t stretch-backports linux-headers-$(uname -r | sed 's/[^-]*-[^-]*-//')
+distribution=$(. /etc/*-release;echo $VERSION_CODENAME)
+apt-get install -t $distribution-backports linux-headers-$(uname -r | sed 's/[^-]*-[^-]*-//')
```

Comment:
Are these commands required on more than stretch?

Reply:
If i'm not mistaken, i had to use them on buster